lib/test_sites/cac.rb

```diff
@@ -16,15 +19,13 @@ def dump_matches
       CSV.open(DataFile.path('cac_comparison.csv'),
                'w',
                write_headers: true,
-               headers: ['Jaro-Winkler distance', 'ours', 'closest CAC match']) do |csv|
+               headers: ['Jaro-Winkler', 'Geo Distance', 'Ours', 'Closest CAC Match']) do |csv|
         closest_matches.each do |match|
           csv << match
         end
       end
     end
 
-    private
```

Comment:
I know this is just a utility class to refine the massaging logic, but could you restore the `private` declaration to where it makes sense? A clear distinction between public and private methods is useful context for anyone new reviewing the code (including me!).